src/cli.rs

```diff
@@ -150,21 +150,21 @@ async fn get_versions(
             MCVersionCLI::Custom(input) => minecraft_versions
                 .into_iter()
                 .find(|v| v.version == input)
-                .context(format!("Could not find Minecraft version {}", input))?,
+                .with_context(|| format!("Could not find Minecraft version {}", input))?,
         },
         match loader_version {
             LoaderVersionCLI::Stable => loader_versions
                 .into_iter()
                 .find(|v| v.version.pre.is_empty())
-                .unwrap(),
+                .with_context(|| "Could not find a stable Quilt Loader version")?,
```

Comment:
this on the other hand should only be `context()` because there is no point in lazily evaluating a constant.

Reply:
Oh 🤦 I was mindlessly copy-pasting and missed that. Usually with this type of mistake clippy catches it, but it didn't this time I think because it's a third-party library.